packages/vx-shape/src/shapes/BarGroupHorizontal.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import cx from 'classnames';
+import { Group } from '@vx/group';
+import Bar from './Bar';
+import { BarGroupProps } from './BarGroup';
+import { ScaleType } from '../types';
+
+type Key = string;
+
+type BarGroupHorizontalProps<Datum> = Pick<
+  BarGroupProps<Datum>,
+  'data' | 'className' | 'top' | 'left' | 'keys' | 'color'
+> & {
+  /** Returns the value (Datum[key]) mapped to the x of a bar */
+  x: (barValue: number) => number;
+  /** Returns the value mapped to the y0 (position of group) of a bar */
+  y0: (d: Datum) => any;
+  /** @vx/scale or d3-scale that takes a key value (Datum[key]) and maps it to an x axis position (width of bar). */
+  xScale: ScaleType;
+  /** @vx/scale or d3-scale that takes a y0 value (position of group) and maps it to a y axis position. */
+  y0Scale: ScaleType;
+  /** @vx/scale or d3-scale that takes a group key and maps it to an y axis position (within a group). */
+  y1Scale: ScaleType;
+  /** Total width of the x-axis. */
+  width: number;
+  /** Override render function which is passed the computed Ba/rGroups. */
+  children?: (barGroups: BarGroup[]) => React.ReactNode;
+};
+
+/** One BarGroup is returned for each datum, which has multiple sub-bars (based on keys). */
+export interface BarGroup {
```

Comment:
could you reuse `BarGroup` type from the `BarGroup.tsx` here?

Reply:
they are slightly different, but I moved to a separate file and was able to consolidate `Bar` definitions (within a `BarGroup`) 👍